### PR TITLE
removed year from copyright headers

### PR DESCRIPTION
**Pull request by royopa**

### CODE REVIEW

File: cron/check_parameters.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
   +----------------------------------------------------------------------+
   | PHP QA GCOV Website                                                  |
   +----------------------------------------------------------------------+
-  | Copyright (c) 2005-2018 The PHP Group                                |
+  | Copyright (c) The PHP Group                                |
```
Comment (petk) [inline cron/check_parameters.php]:
```suggestion
  | Copyright (c) The PHP Group                                          |
```

**Review by royopa — COMMENTED** [review]
corrected!

File: cron/check_parameters.php (hunk 1 above)
Comment (royopa) [reply cron/check_parameters.php]: Corrected the blank spaces

**Comment by petk** [comment]
Thank you @royopa 👍 now, the tricky part where I have no idea who can merge this :D

maybe @nunoplopes or @nikic can help out here a bit?

**Review by petk — APPROVED** [review]

**Comment by nikic** [comment]
Merged as https://github.com/php/web-gcov/commit/80183c679af80626961623c1e7790caf06f1755a.

**Pull request closed without merging**